docs/source/index.rst

```diff
@@ -1,119 +1,135 @@
+It is recommended that you use a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_.
 
-Browser demo
-~~~~~~~~~~~~
+.. note ::
+    .. include:: /versions.rst
+    
+You can also try dissect directly from within your browser:
 
-If you don't feel like doing the above, then there is an interactive browser demo available at
-`try.dissect.tools <https://try.dissect.tools>`_ to play around with!
+`try.dissect.tools <https://try.dissect.tools>`_
+    
 
-You can select evidence files from your local system, such as VMDK or EWF files, and explore some of the capabilities
-of Dissect. Nothing gets uploaded to a server, it all happens locally thanks to `Pyodide <https://pyodide.org/en/stable/>`_!
+Learn more
+----------
 
-.. caution::
+Excited? Great, so are we!
+The next step is to get to know Dissect a little better.
 
-    The browser demo was developed as a quick demo and is bound to be unstable. Not everything may work as expected.
 
-Summary
--------
+* Try the :doc:`/tutorial` to get started!
+* Visit :doc:`/usage/introduction` for an in-depth introduction into Dissect
+* Visit :doc:`/tools/index` for an overview of each tool in the Dissect suite
 
-It's difficult to explain the full capabilities of Dissect in a few sentences, but it can best be summarized as
-an attempt to bring "it just works" to the field of digital forensics. Analysts no longer need to concern themselves
-with thinking about *how* they get artefacts from their source data and can instead focus on *what* artefacts
-they want to analyse.
+Or you can start by taking a look at some community articles and videos:
+:doc:`/resources/dissect-in-action` or
+:doc:`/resources/talks-and-conferences` to begin with.
 
-For example, with Dissect you can:
+Get in touch, join us on `github <https://github.com/fox-it/dissect.target>`_!
 
-* Make a timeline of the event log from all investigation data at once;
-* Identify anomalies in artefacts such as services, tasks and run keys over a large amount of investigation data;
-* Perform incident response on partially ransomwared (encrypted) virtual machine disks;
-* Perform complex IOC checks on thousands of hosts in a couple of hours;
-* Export all USN journal records from a Bitlocker-encrypted disk to Splunk without waiting for decryption;
-* Collect forensic artefacts from all live virtual machines directly from the hypervisor with no down time;
-* Export all artefacts easily to any data format you want, for example CSV, JSON or Avro, or stream directly to Splunk or Elastic;
-* ... and much more.
 
-For more information about what Dissect is and how it works, read on at :doc:`/overview/index`.
+.. toctree::
+    :hidden:
+    
+    Home <self>
 
 .. toctree::
+    :caption: Basics
     :hidden:
 
-    Home <self>
-    /overview/index
+    Install </install>
+    Tutorial </tutorial>
+    Querying </target-query>
+    Shell </target-shell>
+    Acquire </acquire>
+    RDump </rdump>
```

Comment:
```suggestion
    rdump </rdump>
```

Reply:
Title case is more consistent here.

Reply:
Please show me where rdump was ever written as RDump.